src/app/modules/timeline-config/services/timeline-global-form-builder/timeline-global-form-builder.service.ts

```diff
@@ -76,13 +103,38 @@ export class TimelineGlobalFormGroup extends ConfigFormGroup {
                 150,
                 1
               )
-            },
-              {
-                dependsOn: () => [this.customControls.useDetailedTimeline],
-                onDependencyChange: (control) =>
-                  control.enableIf(this.customControls.useDetailedTimeline.value)
+            }, {
+              dependsOn: () => [this.customControls.useDetailedTimeline],
+              onDependencyChange: (control) =>
+                control.enableIf(this.customControls.useDetailedTimeline.value)
+            }
+            ).withTitle(marker('Detailed timeline')),
+            additionalCollections: new ConfigFormGroup({
+              collections: new MultipleSelectFormControl(
+                '',
+                marker('Additional collections'),
+                marker('Additional collections description'),
+                false,
+                collectionService.getCollections().filter(c => c !== collection).map(c => ({ label: c, value: c })),
+                {
+                  optional: true,
+                },
+                false
+              )
+            }, {
+              dependsOn: () => [this.customControls.tabsContainer.dataStep.timeline.collection],
+              onDependencyChange: () => {
+                if (this.customControls.tabsContainer.dataStep.timeline.collection.dirty) {
+                  this.customControls.tabsContainer.dataStep.additionalCollections.collections.savedItems = new Set<string>();
```

Comment:
just to be sure, this condition is needed for when we initialise the form from an existing dahsboard ?

Reply:
Yep, it's needed to reset selected collections only when the user change the main collection and not when the form is initialised